packages/errors/src/messages.ts

```diff
@@ -391,6 +393,10 @@ export const SolanaErrorMessages: Readonly<{
     [SOLANA_ERROR__INSTRUCTION_ERROR__UNKNOWN]: '',
     [SOLANA_ERROR__INSTRUCTION_ERROR__UNSUPPORTED_PROGRAM_ID]: 'Unsupported program id',
     [SOLANA_ERROR__INSTRUCTION_ERROR__UNSUPPORTED_SYSVAR]: 'Unsupported sysvar',
+    [SOLANA_ERROR__INSTRUCTION_PLANS__CANNOT_PACK_USING_PROVIDED_MESSAGE]:
+        'Cannot pack the next instructions using the provided message.',
```

Comment:
In all cases, this is because it won't fit, right?
```suggestion
    [SOLANA_ERROR__INSTRUCTION_PLANS__MESSAGE_CANNOT_ACCOMMODATE_PLAN]:
        'The provided message has insufficient capacity to accommodate the next instruction(s) in this plan',
```